### ISSUE TITLE
"Tried to load angular more than once."

### ISSUE THREAD

**Issue by szepeviktor** [formio/ngFormio#186]
Hello!

formio-complete.min.js:53 WARNING: Tried to load angular more than once.

This is my "app":

``` html
    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
    <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular.min.js"></script>
    <script type="text/javascript" src="//npmcdn.com/ng-formio@latest/dist/formio-complete.min.js"></script>
</head>
<body>
    <div id="main">
        <p>Content.</p>

        <formio style="outline: 2px solid green;" src="'https://gyxqbhairwmxoyw.form.io/contact'"></formio>
        <script type="text/javascript">
            angular.module('formioApp', ['formio']);
            angular.bootstrap(document, ['formioApp']);
        </script>
```

**Comment by szepeviktor** [comment]
formio-complete.js:41594 WARNING: Tried to load angular more than once.

**Comment by szepeviktor** [comment]
@travist Or: formio-complete.js:41594 WARNING: Tried to load angular more than once.

**Comment by travist** [comment]
We will take a look. Thanks.

**Comment by laradevitt** [comment]
Hello,

I'm running into this, too. Unless I'm seeing things, looks to me like formio-complete.js ([2.2.4](https://raw.githubusercontent.com/formio/ngFormio/v2.2.4/dist/formio-complete.js)) does contain Angular when the [docs say it shouldn't](https://github.com/formio/ngFormio/blob/master/README.md).

**Comment by Liam-Ryan** [comment]
Still seeing this on latest build

**Comment by ghachemsaif** [comment]
No solution yet for this issue please ??

**Comment by rmckeel** [comment]
A few notes here, as I just had to push around this.

What's interesting here is that https://github.com/formio/ngFormio/blob/master/src/formio-complete.js does *not* include angular, while https://github.com/formio/ngFormio/blob/master/src/formio-full.js clearly does.  Perhaps something in the build script is injecting Angular automatically because it is a dependency of libraries like `angular-sanitize`?  Just a guess.

I have an NG1 (TypeScript) app, and I ended up putting the includes from /src/formio-complete.js manually into the file.  That prevented & fixed the 'loading angular multiple times' issue.  (NOTE: that may also require some `npm i` for each library you are trying to require.

That's not super convenient, however may be a temporary workaround.
